resources/public/api/conf/1.0/application.raml

```diff
@@ -440,11 +440,11 @@ uses:
                     code: INVALID_REQUEST
   /organisations/corporation-tax/{crn}/company/details:
     uriParameters:
-      utr:
-        description: "A Unique Tax Reference"
+      crn:
```

Comment:
This is not a patch right? For the COTAX (CRN) endpoint we want a `CRN` not a `UTR`... Right?

Reply:
Yeah, this change is more of a correction than a patch